tests/test_generators.py

```diff
@@ -164,6 +164,28 @@ def testWeightedAdjacency(self):
         self.assertTrue(el == [(0,1), (0,2), (1,0), (3,1)])
         self.assertTrue(g.es["w0"] == [1, 2, 2, 1])
 
+    def testDataFrame(self):
```

Comment:
Use the `skipIf()` decorator from `tests/utils.py` to skip the test case when `pandas` is not installed. See `tests/test_basic.py` for inspiration; there is a test case that is skipped when NumPy is not installed.

Reply:
:+1: